k4e-agent.spec

```diff
@@ -0,0 +1,33 @@
+%define _build_id_links none
+
+Name:       k4e-agent
+Version:    1.0
+Release:    1%{?dist}
+Summary:    Agent application for the K4E (Kubernetes for Edge) solution
+
+Group:      K4E
+License:    GPL
+Source0:    https://github.com/jakub-dzon/%{name}-%{version}.tar.gz
+
+BuildRequires:  golang
```

Comment:
For RHEL we need few more dependencies:
* [gpgme-devel](http://mirror.centos.org/centos/8/PowerTools/x86_64/os/Packages/gpgme-devel-1.13.1-7.el8.x86_64.rpm)
*  [btrfs-progs-devel](https://mirror.rackspace.com/elrepo/testing/el8/x86_64/RPMS/btrfs-progs-devel-4.20.2-1.el8.x86_64.rpm)
* [btrfs-progs](https://mirror.rackspace.com/elrepo/testing/el8/x86_64/RPMS/btrfs-progs-4.20.2-1.el8.x86_64.rpm)
* [libbtrfs](https://mirror.rackspace.com/elrepo/testing/el8/x86_64/RPMS/libbtrfs-4.20.2-1.el8.x86_64.rpm)
* [libbtrfsutil](https://mirror.rackspace.com/elrepo/testing/el8/x86_64/RPMS/libbtrfsutil-4.20.2-1.el8.x86_64.rpm)
* [device-mapper-devel](http://mirror.centos.org/centos/8/PowerTools/x86_64/os/Packages/device-mapper-devel-1.02.175-5.el8.x86_64.rpm)
* [libassuan-devel](http://mirror.centos.org/centos/8/PowerTools/x86_64/os/Packages/libassuan-devel-2.5.1-3.el8.x86_64.rpm)

I haven't decide to update this yet